src/eslint-config-adeira/ourRules.js

```diff
@@ -275,6 +275,7 @@ module.exports = ({
   'flowtype/no-dupe-keys': ERROR,
   'flowtype/no-existential-type': ERROR, // https://github.com/facebook/flow/issues/6308
   'flowtype/no-flow-fix-me-comments': OFF,
+  'flowtype/no-internal-flow-type': WARN,
```

Comment:
Should be `'flowtype/no-internal-flow-type': NEXT_VERSION_ERROR` when you remove `adeira/no-internal-flow-type`.